prism-languages.php

```diff
@@ -63,4 +62,5 @@ function mkaz_code_syntax_block_get_supported_languages() {
 	 * @param string $languages Array of languages supported
 	 */
 	return apply_filters( 'mkaz_code_syntax_language_list', $languages );
+	// Test PHPCS check, should give warning for this comment
```

Comment:
:no_entry_sign: **Error**: Inline comments must end in full-stops, exclamation marks, or question marks (*Squiz.Commenting.InlineComment.InvalidEndChar*).